benchmark/performanceTest/main.go

```diff
@@ -29,22 +28,30 @@ import (
 )
 
 var (
-	outputDir = "./output"
+	outputDir = "../performanceTest/output"
 )
 
 func main() {
-	parseFileAccessPatterns := flag.Bool("parseFileAccess", false, "Parse fuse file access patterns.")
+	var numberOfTests int
+	var configCsv string
+	var showCom bool
+	var parseFileAccessPatterns bool
+	var commit string
+
+	flag.BoolVar(&parseFileAccessPatterns, "parseFileAccess", false, "Parse fuse file access patterns.")
```

Comment:
Doesn't have to be a part of this PR but maybe we can remove the common initialization steps between both binaries into another package.

Reply:
Will add this in the next milestone along with the CI pipeline